src/Sentry/SentryEvent.cs

```diff
@@ -290,7 +290,7 @@ internal static SentryEvent FromJson(JsonElement json, Exception? exception)
     {
         var modules = json.GetPropertyOrNull("modules")?.GetStringDictionaryOrNull();
         var eventId = json.GetPropertyOrNull("event_id")?.Pipe(SentryId.FromJson) ?? SentryId.Empty;
-        var timestamp = json.GetPropertyOrNull("timestamp")?.GetDateTimeOffset();
+        var timestamp = json.GetSafeDateTimeOffset("timestamp"); // Native sentryevents are serialized to epoch timestamps
```

Comment:
Given this is a FromJson method, are we sure it will always be reading JSON that's been serialised by the native SDK? Might it not also be used to read JSON serialised by the managed SDK?

Reply:
I handle both the native and managed cases in that new method.  This method was being used by the native deserialization calls.

Reply:
Shouldn't we handle both here as well? It's a public method.

Reply:
GetSafeDateTimeOffset handles both epoch timestamp and standard string based datetimes